cli_tools/gce_vm_image_import/importer/bootable_disk_processor.go

```diff
@@ -37,8 +37,8 @@ type bootableDiskProcessor struct {
 	OS              string
 }
 
-func (b bootableDiskProcessor) process(ctx context.Context) (err error) {
-	err = b.workflow.RunWithModifiers(ctx, b.preValidateFunc(), b.postValidateFunc())
+func (b *bootableDiskProcessor) process(ctx context.Context) (err error) {
```

Comment:
This package is written to follow the guidance on contexts, specifically that callees are responsible for watching for cancellations. This PR, however, deviates from that guidance, but leaves the package in a half-way state where contexts are passed around, but aren't used. Recall the confusion caused where Daisy accepts a context, but you're not allowed to cancel it. Since Daisy doesn't follow the context idiom, it's hard for future users to understand and use it. By following idiomatic guidance, I'm hoping to avoid that type of confusion.

I can see two approaches forward:
1. Move timeout logic fully into importer.go, and remove context from the signatures of downstream functions.
2. Keep the current architecture (see #1273)

I see two downsides with (1). First, it breaks with idiomatic patterns. So, if we do that, I think we should make a strong argument for why the idioms shouldn't apply here. Second, the change will have a large blast radius, so we'll need to justify that cost.

Reply:
What do you mean by large blast radius in case of removing context from inflater/processors? The change will be contained within the package.

Reply:
You're right, it's all in the same package. I just meant that the heavy-lifting interfaces (validation, inflation, and processing) would all need to be changed, along with their call sites and tests.